### PR TITLE
Localization handback

### PR DESCRIPTION
**Pull request by srivatsn**
This is the loc handback for strings that were added recently.

### CODE REVIEW

**Comment by nguerrera** [comment]
@srivatsn, we need to run the tool on this so that translated resx/vsct/xaml get updated. This step will go away when @tmat's changes (incorporating repo toolset + xliff tasks) are merged.

**Comment by srivatsn** [comment]
Done

**Comment by nguerrera** [comment]
**Customer scenario**

Some project system text will be untranslated for non-English VS users without absorbing this handback.


**Bugs this fixes:** 

[VSO 464718](https://devdiv.visualstudio.com/DevDiv/_workitems?_a=edit&newQuery=true&id=464718&triage=true)


**Workarounds, if any**

None

**Risk**

Low

**Is this a regression from a previous update?**

No

**Root cause analysis:**

N/A. Standard OSS loc process

**How was the bug found?**

N/A

**Comment by nguerrera** [comment]
@Pilchie

File: src/Microsoft.VisualStudio.Editors/AddImportsDialogs/AddImports.ja.resx [hunk 1]
```diff
@@ -127,6 +127,6 @@
     <value>'{0}' をインポートせず、'{1}' を '{2}' に変更する。(&amp;D)</value>
   </data>
   <data name="AddImportsExtensionMethodsMainFormatString" xml:space="preserve">
-    <value>'{0}' をインポートすると、拡張メソッドの呼び出しを含めて、ファイル内の他のコードの意味が変わる場合があります。'{1}' を '{2}' に変更して修正するには、[OK] をクリックします。 </value>
+    <value>{0}' をインポートすると、拡張メソッドの呼び出しを含めて、ファイル内の他のコードの意味が変わります。'{1}' を '{2}' に変更して修正するには、[OK] をクリックします。 </value>
```
Comment (SolderedMushroom) [inline src/Microsoft.VisualStudio.Editors/AddImportsDialogs/AddImports.ja.resx]: Was just the opening apostrophe meant to be removed?

**Pull request merged**